e2e/tests/home/home-page.spec.ts

```diff
@@ -38,19 +37,34 @@ describe('Home page ui tests', () => {
       const cardElem = new BaseElement(page, card.asElementHandle());
       expect(await cardElem.isVisible()).toBe(true);
       const size = await cardElem.getSize();
-      if (width === undefined) {
+      expect(size).toBeTruthy();
+      expect(size.height).toBeGreaterThan(1);
+      expect(size.width).toBeGreaterThan(1);
+
+      if (width === undefined || height === undefined) {
         width = size.width; // Initialize width and height with first card element's size, compare with rest cards
         height = size.height;
       } else {
         expect(size.height).toEqual(height);
         expect(size.width).toEqual(width);
       }
+
+      // check workspace name has characters
+      const cardName = await card.getResourceCardName();
+      expect(cardName).toMatch(new RegExp(/^[a-zA-Z]+/));
+
+      // check ellipsis icon existed
+      expect(await card.getEllipsisIcon()).toBeTruthy();
+
+      // Assumption: test user is workspace Owner.
+      // Check ellipsis popup has right option values
+      const links = await card.getPopupLinkTextsArray();
+      expect(links).toEqual(expect.arrayContaining(['Share', 'Edit', 'Duplicate', 'Delete']));
```

Comment:
relocated steps from `'Check Workspace card on Home page'` to this test. Change doesn't affect test coverage or code logic.